source/common/http/http2/codec_impl.cc

```diff
@@ -755,7 +755,7 @@ Status ConnectionImpl::onFrameReceived(const nghttp2_frame* frame) {
     // to match the ping and ack.
     uint64_t data;
     static_assert(sizeof(data) == sizeof(frame->ping.opaque_data), "Sizes are equal");
-    memcpy(&data, frame->ping.opaque_data, sizeof(data));
+    memcpy(&data, frame->ping.opaque_data, sizeof(data)); // NOLINT(safe-memcpy)
```

Comment:
You can drop the static_assert above and replace these two lines with `SAFE_MEMCPY(&data, frame->ping.opaque_data)`.